.changeset/sweet-parents-chew.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/lab": patch
+---
+
+Fix tabs container height to fit its contents and display as block so it can take 100% width without an extra wrapper
```

Comment:
```suggestion
- Fixed tabs container height to fit its contents and display as block so it can take 100% width without an extra wrapper.
- Updated stories to reflect up to date implementation.
```